src/api/app/views/shared/_requests.html.erb

```diff
@@ -3,7 +3,7 @@
   request_table_id ||= 'request_table'
   hide_state ||= false
   no_target ||= false
-  paginate = serverProcessing ? true : requests.count > 25
+  paginate = serverProcessing || (!requests.nil? && requests.count > 25)
```

Comment:
ah, I meant relations.count.to_i. Thinking about it, can a has many relation be nil in rails or would it be an empty array?